src/rgw/rgw_kafka.cc

```diff
@@ -107,7 +107,7 @@ struct connection_t {
   // ctor for setting immutable values
   connection_t(CephContext* _cct, const std::string& _broker, bool _use_ssl, bool _verify_ssl, 
           const boost::optional<const std::string&>& _ca_location,
-          const std::string& _user, const std::string& _password) :
+          const std::string& _user, const std::string& _password) : status(STATUS_OK),
```

Comment:
siunce you're already using member initializers for other stuff:
```diff
   uint64_t delivery_tag = 1;
-  int status;
+  int status = STATUS_OK;
   mutable std::atomic<int> ref_count = 0;
```

Reply:
fixed